index.html

```diff
@@ -115,6 +115,7 @@ <h3><a id="tools" href="#tools">Tools and plugins</a></h3>
             <li><a href="https://jefftriplett.com/">Jeff Triplett</a> gives us a nice <a href="https://www.djangosnippets.org/snippets/1421/">Django Snippet</a> for you Python folk.</li>
             <li>Drupal users, <a href="https://www.drupal.org/project/naked">get the module</a>!</li>
             <li>If you are game you can use <a href="https://weblog.200ok.com.au/">Ben Buchanan</a>’s <a href="https://bitbucket.org/snippets/200ok/dKpk">JavaScript snippet</a>. Works anywhere, removes everything, buyer beware!</li>
+            <li>If you are a Gopher, check out the <a href="https://github.com/omz13/today-go">today-go</a> package by <a href="https://omz13.com/>David Somers</a>.</li>
```

Comment:
Please add closing quotes on second link.

Reply:
gah! sorry about that... editing in directly in GitHub had a hiccup... closing quote now added.

Technically, yes, a general anniversary tool, but with helpers for CssNaked, et al.. The readme has a pointer to the docs that are generated on pkg.go.dev, and I've added more to package.go which gets automatically copied into the readme.

Reply:
Thanks for updating and clarifying!